diff --git a/texture_baker/texture_baker/csrc/baker_kernel.cu b/texture_baker/texture_baker/csrc/baker_kernel.cu
--- a/texture_baker/texture_baker/csrc/baker_kernel.cu
+++ b/texture_baker/texture_baker/csrc/baker_kernel.cu
@@ -1,6 +1,6 @@
 #include <ATen/ATen.h>
 #include <ATen/Context.h>
 #include <ATen/cuda/CUDAContext.h>
-#include <torch/extension.h>
+#include <torch/types.h>

 #include "baker.h"
